docs/src/mui/Table.tsx

```diff
@@ -0,0 +1,45 @@
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+
+function createData(name: string, price: number) {
+  return { name, price };
+}
+
+const rows = [
```

Comment:
Maybe we could write only a array with an object with element of name and price like `{name: "xxx", price: 2}`, and we don't need the mapper function createData :)

Reply:
That would work as well.  If I have time I can update it to use this approach.  This story is an adjusted version of an MUI table example, which is where the createData function came from.